mips/ebase.S

```diff
@@ -63,7 +68,6 @@ maybe_hit:
         # it's valid! We need to check it.
 
         tlbr                    # Read the entry with matching VPN2
-        ehb
```

Comment:
The `ehb` instruction still needs to be there, preferably just before the consumers, that is before the `mfc0 k1, C0_ENTRYLO1`. It shouldn't be removed completely.

Reply:
Right! I restored `ehb` at the point. Let's not move it around for clarity's sake. Two cycles are not worth it for now.